test/autorest/lrogroup/lros_test.go

```diff
@@ -69,1142 +72,974 @@ func TestLROBeginDelete202NoRetry204(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	for !poller.Done() {
-		_, err := poller.Poll(context.Background())
-		if err != nil {
-			t.Fatal(err)
-		}
-		time.Sleep(200 * time.Millisecond)
-	}
-	resp, err = poller.FinalResponse(context.Background())
+	resp, err = resp.PollUntilDone(context.Background(), time.Duration(1)*time.Millisecond)
```

Comment:
Nit, casting `1` to `time.Duration` is unnecessary.

Reply:
Done